src/EntryPoints/BitbucketFunction.php

```diff
@@ -22,7 +23,11 @@ public function handleParserFunctionCall( Parser $parser, string ...$arguments )
 			new CategoryUsageTracker( $parser )
 		);
 
-		$useCase = EmbedExtensionFactory::getInstance()->newEmbedUseCaseForBitbucketFunction( $presenter );
+		$renderer = ( new ContentRendererFactory() )->createContentRenderer( array_slice( $arguments, 1 ) );
+		$parser->getOutput()->addModules( $renderer->getOutputModules() );
+		$parser->getOutput()->addModuleStyles( $renderer->getOutputModuleStyles() );
```

Comment:
`addModules` and `addModuleStyles` are separate because the latter loads CSS immediately (as part of page CSS bundle), whereas the former loads it at along with the JS. Since these syntax highlighters tend to be heavy, this causes a slight delay in the CSS.

There is some initial styling differences between MW and Prism for the code box (`<pre>`/`<code>` border, background, position), so having the CSS load earlier makes the initial box more similar, excluding the actual syntax highlighting.